common/persistence/sql/sqlplugin/mysql/plugin.go

```diff
@@ -47,7 +41,7 @@ import (
 const (
 	// PluginName is the name of the plugin
 	PluginName                   = "mysql"
-	dsnFmt                       = "%s:%s@%v(%v)/%s"
+	dsnFmt                       = "%v:%v@%v(%v)/%v"
```

Comment:
what is the point of having this as separate const?

Reply:
do you mean this?
```go
const dsnFmt = "%v:%v@%v(%v)/%v"
```

usually, all string const are defined at the very top